app/assets/stylesheets/fortitude/components/_flag.scss

```diff
@@ -25,13 +26,15 @@ Class            | Description
     <img src="//placehold.it/150" alt="placeholder" />
   </div>
   <div class="flag__body">
-    Lorem ipsum dolor sit amet,
-    consectetur adipisicing elit.
-    Iste magnam facilis quas,
-    eos unde illum rem dicta.
-    Atque repudiandae,
-    numquam laudantium quas voluptas,
-    facilis? Illo excepturi nihil quidem a esse!
+    <p class="xs-pl1">
```

Comment:
That didn't work, unfortunately. Good call on handling responsive margin / padding.